pkg/util/proc/process_unix.go

```
@@ -135,6 +135,9 @@ func findProcessSearch(binary, cmdlinePattern string) (*UnixProcess, error) {
 func processes() ([]*UnixProcess, error) {
 	d, err := os.Open("/proc")
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, nil
```

could that fail on linux systems? seems to ignore the error silently.